include/overlay_01_02200858.h

```diff
@@ -1,7 +1,7 @@
 #ifndef POKEHEARTGOLD_OVERLAY_01_02200858_H
 #define POKEHEARTGOLD_OVERLAY_01_02200858_H
 
-#include "field_player_avatar.h"
+#include "player_avatar.h"
```

Comment:
put in field folder ?

Reply:
imo since it's not in a field overlay it shouldn't be in the field folder